userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -700,22 +717,22 @@ bool docker_async_source::parse_docker(std::string &container_id, sinsp_containe
 			// separate thread so this is ok.
 			g_logger.format(sinsp_logger::SEV_DEBUG,
 					"docker_async (%s), secondary (%s): Doing blocking fetch of secondary container",
-					container_id.c_str(),
+					instruction.container_id.c_str(),
 					secondary_container_id.c_str());
 
-			if (parse_docker(secondary_container_id, pcnt))
+			if(parse_docker(docker_async_instruction(secondary_container_id, instruction.request_rw_size), pcnt))
```

Comment:
We only need the IP from the secondary fetch so you should be able to always pass `false` here.

Reply:
thanks